huxley/core/admin/assignment.py

```diff
@@ -39,30 +41,69 @@ def load(self, request):
         '''Loads new Assignments.'''
         assignments = request.FILES
         reader = csv.reader(assignments['csv'])
+        successful_rows = []
```

Comment:
So instead of keeping track of successful rows I would try keeping the generator that is returned from `generate_assignments` and instead put a try catch in there so that every time an error occurs we catch it, append the row to `failed_rows` and then continue on in our iteration